src/formatDate.js

```diff
@@ -7,8 +7,33 @@
  *
  * @returns {string}
  */
-function formatDate(date, fromFormat, toFormat) {
-  // write code here
+function formatDate(date, fromFormat, toFormat, cutoffYear = 30) {
```

Comment:
Consider adding validation to ensure that the `date` string matches the `fromFormat`. This will help prevent incorrect conversions or errors if the input format is not as expected.